client/verta/verta/client.py

```diff
@@ -1736,13 +1735,3 @@ def find_datasets(
             datasets = datasets.find(predicates)
 
         return datasets
-
-    def _create_organization(
-        self, name, desc=None, collaborator_type=None, global_can_deploy=None
-    ):
-        return Organization._create(
-            self._conn, name, desc, collaborator_type, global_can_deploy
-        )
```

Comment:
Unfortunately, I realize that a lot of our tests right now depend on creating custom workspace during tests 🤔 

I will need to talk to Elizabeth about whether we can remove these specific Python tests in the future, but for now I think maybe we need a `_create_workspace()`, `Workspace.add_member()` and `Workspace.delete()` just for the tests

Reply:
Is it necessary to create a workspace during test? Why is it a problem to use existing workspace?

It is possible to create a workspace during test, but the problem is that we don't have workspace.add_member. Members are added to workspaces using groups. So the problem is that if we create workspaces in test we also need to call endpoints to access or create groups, roles, users and organizations

Reply:
Also please note that in permission v2 we can't set individual permission by resource id. The system is simplified comparing to v1 so now admin can set permission to use resource types by groups of users in specified workspaces.
So as I see some of permission tests in Python Client can't be migrated to v2.

Reply:
> So as I see some of permission tests in Python Client can't be migrated to v2.

Yes, we will need to end up deleting tests that are no longer applicable to v2 😞